lib/auth-header.js

```diff
@@ -0,0 +1,14 @@
+const replace = require('./replace-vars');
+
+module.exports = ({scheme, token, username, password}) => {
+  console.log('authHeader', scheme, username);
```

Comment:
😬

Reply:
![nothing to see here](https://media.giphy.com/media/3ohs7KViF6rA4aan5u/giphy.gif)